packages/strace/project.bri

```diff
@@ -26,7 +26,7 @@ export default function strace(): std.Recipe<std.Directory> {
         ;;
     esac
 
-    make -j16
+    make -j "$(nproc)"
```

Comment:
Just to notice here, I didn't set the env variable OMP_NUM_THREADS to override nproc command output. I think it's not necessary for this package.